UNITTESTS/stubs/rtx_mutex_stub.c

```diff
@@ -0,0 +1,11 @@
+#include "cmsis_os2.h"
```

Comment:
code files need license on top of them, please add to all new files

Reply:
Added license where it was missing.